src/components/right-panel/right-panel.js

```diff
@@ -0,0 +1,92 @@
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
```

Comment:
can be done later Most of the JSX is repeating you can create array of object and map over it